nipype/pipeline/plugins/tests/test_callback.py

```diff
@@ -61,3 +61,37 @@ def test_callback_exception(tmpdir, plugin, stop_on_first_crash):
 
     sleep(0.5)  # Wait for callback to be called (python 2.7)
     assert so.statuses == [("f_node", "start"), ("f_node", "exception")]
+
+
+@pytest.mark.parametrize("plugin", ["Linear", "MultiProc", "LegacyMultiProc"])
```

Comment:
This test fails if `pandas` is not installed. Since it's not a hard requirement, we need something like:

```suggestion
@pytest.mark.parametrize("plugin", ["Linear", "MultiProc", "LegacyMultiProc"])
@pytest.mark.skipif(not has_pandas, "Test requires pandas")
```

And at the top:

```Python
try:
    import pandas
    has_pandas = True
except ImportError:
     has_pandas = False
```